### ISSUE TITLE
Expression-based contract syntax

### ISSUE THREAD

**Issue by the-Mitu** [intellij-dlanguage/intellij-dlanguage#418]
D 2.081 is to be released in the beginning of July and is going to introduce a new contract syntax: https://dlang.org/changelog/pending.html#expression-based_contract_syntax

It would be nice to have it implemented by then.

**Comment by the-Mitu** [comment]
Let's note that D 2.081 is already out. :)

**Comment by pirocks** [comment]
Waiting on https://github.com/dlang-community/libdparse/issues/218.

**Comment by the-Mitu** [comment]
dlang-community/libdparse#218 already closed.

**Comment by andy-hanson** [comment]
I think this is the same error?
```d
int abs(int x)
out (res; res >= 0) {
	return x < 0 ? -x : x;
}
```
Shows an error at `res;`: `Expected ) instead of ;`
